tests/Helper/ErrorFunctions.php

```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types=1);
+
+namespace Starlit\Db {
+    class ErrorStackTestHelper
+    {
+        public static $errors = [];
+    }
+
+    function trigger_error(string $error_msg, $type = E_USER_NOTICE)
+    {
+        ErrorStackTestHelper::$errors[] = [
+            'message' => $error_msg,
+            'type' => $type
+        ];
+    }
+
+    function usleed(int $start, int $stop)
+    {
+
+    }
```

Comment:
What's this?

Reply:
hahaha, good question. Guess this is related to testing to override global functions in a custom namespace, but will have to check 👍

Reply:
wrongly spelled and not used. Removed 😆